markdown/ch4.markdown

```diff
@@ -84,6 +86,8 @@ ax.set_xlabel('Time [s]')
 ax.set_ylabel('Signal amplitude');
 ```
 
```

Comment:
@hdashnow I did but in the interest of expedience, I'll make the changes myself this time. =)